P5/Source/Guidelines/en/SA-LinkingSegmentationAlignment.xml

```diff
@@ -3242,41 +3199,130 @@ will only select the text of the poem, with no markup inside.</p>
 </div>
 </div>
 
-<div type="div2" xml:id="SASOstdf">
-  <!-- head>Placement of Stand-off Markup and Contextual Information</head -->
-  <head>The <gi>standOff</gi> Container</head>
-  <p>The <gi>standOff</gi> element is a catch-all element intended for
-  holding content that is neither transcribed from the source
-  document, nor strictly speaking metadata about said document.
-  Examples include contextual information about named entities
-  (typically encoded using <gi>listBibl</gi>, <gi>listOrg</gi>,
-  <gi>listNym</gi>, <gi>listPerson</gi>, or <gi>listPlace</gi>),
-  annotations indicating the morphosyntactic features of a text,
-  collections of <gi>alt</gi> elements for encoding ambiguity of
-  reference, and an encoded <gi>castList</gi> used to record
-  information about speaking characters in a drama who were not
-  mentioned in the source cast list.</p>
-  <p>The <gi>standOff</gi> element may also be used to hold
-  transcribed text that does not, for some reason, fit into the
-  encoding model being used. Examples include encoding source
-  authorial footnotes in the <gi>standOff</gi> in order to more easily
-  represent source pages when a footnote runs across a page break, or
-  to encode words that appear in the source text as part of an
-  acrostic<!-- (using <gi>acrostic</gi> in <gi>extraTextual</gi>)-->.
-  <specList>
-    <specDesc key="standOff"/>
-  </specList>
-  </p>
-  <p>As a member of <ident type="class">model.resource</ident>,
-  <gi>standOff</gi> may occur as a child of <gi>TEI</gi> (or
-  <gi>teiCorpus</gi>). If the metadata that describes the
-  <gi>standOff</gi> is largely the same as the metadata that describes
-  the associated resource (e.g., the transcribed text in
-  <gi>text</gi>), then the <gi>standOff</gi> and the encoded
-  associated resource may appear as children of the same <gi>TEI</gi>
-  element.</p>
-  <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/standOff.xml"/>
-</div>
+  <div type="div2" xml:id="SASOstdf">
+    <!-- head>Placement of Stand-off Markup and Contextual Information</head -->
+    <head>The <gi>standOff</gi> Container and Annotations</head>
+    <p>The <gi>standOff</gi> element is intended to hold content that is neither transcribed from
+      the source document, nor strictly speaking metadata about said document. Examples include
+      contextual information about named entities (typically encoded using <gi>listBibl</gi>,
+        <gi>listOrg</gi>, <gi>listNym</gi>, <gi>listPerson</gi>, or <gi>listPlace</gi>), annotations
+      indicating the morphosyntactic features of a text, and annotations commenting on or
+      associating parts of a text with additional information. <specList>
+        <specDesc key="standOff"/>
```

Comment:
I really don't like those introduced here. They seem like weird edge cases to me, even though they're legal. The "alt" stuff seems like something probably no-one uses (I'd generally use `<choice>`). The castList thing strikes me as quite bizarre.

Reply:
Well we need to find another reasonable place then (or leave them here), because edge cases they are not. (E.g., we have 84 texts in which there are characters in a drama not mentioned in the printed cast list, i.e. that would make use of `standOff/castList`. That’s in 427 texts, or almost 20% of them.) Far fewer acrostics, admittedly: only 10, or 2.3% of texts. (Interestingly, at least to me, 3 from late 16th, 3 from 17th, 2 from late 18th, 1 from very early 19th, and 1 from early 19th centuries.)

And while I think you are right, `<alt>` is very rarely used, I submit that a) this is in part because TEI has not given good guidance on, among other things, where to put them; and b) they are still very useful, and in particular useful for cases for which neither `<choice>` nor `@exclude` is really a viable option. For a pre-`<standOff>` example see [the slide on reference to ambiguous name](https://www.wwp.neu.edu/outreach/seminars/_current/presentations/contextual_encoding/advanced_context_09.xhtml) from a WWP workshop.

Reply:
I fail to see why you wouldn't just supplement the cast list in the text if you wanted to, perhaps noting the entries that were missing in the original (I would use `<supplied>`), but in any case, I think those things belong in the discussion about the elements in question.

Any chance we could kill all of the mad examples of `<alt>` in the GLs and replace them with your perfectly clear and sane one?

Reply:
> I fail to see why you wouldn't just supplement the cast list in the text …

Well, you know me well enough to know that **I** would be OK with such a solution if it were possible. Your particular suggestion is not, as `<supplied>` as not permitted in the content of `<castList>`. However, I personally would be OK using `@type="supplied"` on `<castList>` or `<castItem>`s. But the _Guidelines_ aren’t for just me, and it is a perfectly reasonable editorial encoding policy to say “we transcribe only what is actually in the source document into `<text>` (or `<sourceDoc>`) — extra stuff like corrections, supplied contextual information, annotations, results of acrostics, etc. all go elsewhere”. And now that we have `<standOff>`, there is such an elsewhere.

> Any chance we could kill all of the mad examples of <alt> in the GLs and replace them with your perfectly clear and sane one?

Dunno. Might be hard to kill _all_ of the existing examples. But certainly adding this one would be a good idea, IMHO.

Reply:
I don't think this is a blocker for accepting the PR, so what I want to do is make an issue or two covering out-of-band cast lists and rationalizing the examples for alt/altGrp.